client/src/pages/admin/Index.js

```diff
@@ -113,10 +217,46 @@ const BaseAdminIndex = ({ pageDispatchers, loadAppData }) => {
     refetch()
   }
 
-  function save(values, form) {
-    // settings as JSON
-    const settings = Object.map(values, (key, value) => ({ key, value }))
-    return API.mutation(GQL_SAVE_ADMIN_SETTINGS, { settings })
+  function clearCache() {
+    setActionLoading(true)
+    return API.query(CLEAR_CACHE, {})
+      .then(result => toast.success(result?.clearCache))
+      .catch(handleError)
+      .finally(() => setActionLoading(false))
+  }
+
+  function reloadDictionary() {
+    setActionLoading(true)
+    return API.query(RELOAD_DICTIONARY, {})
+      .then(result => toast.success(result?.reloadDictionary))
+      .catch(handleError)
+      .finally(() => setActionLoading(false))
+  }
+
+  function loadUserActivitiesAndRecentUsers() {
+    setActionLoading(true)
+    return API.query(USER_ACTIVITIES, {})
+      .then(data => {
+        const recentCalls = data?.userActivities?.recentCalls || []
+        recentCalls.forEach(ua => (ua.uuid = uuidv4()))
```

Comment:
we should get the real uuid from the graphql

Reply:
Since this data is not coming from database. It doesn't have a uuid. We need a stable identity in front-end to be used as Key by react. That's why I am generating it on the client when the data is received. I think the name `uuid` is confusing so I changed it to `listKey` and added a comment to clarify.